tests/dashbio_demos/app_molecule3d.py

```diff
@@ -64,80 +156,86 @@ def layout():
-                                        ),
+                                        html.Div(className='app-controls-name',
+                                                 children='Select structure'),
+
                                         dcc.Dropdown(
                                             id='dropdown-demostr',
                                             options=[
                                                 {
-                                                    'label': 'Protein',
-                                                    'value': '{}3aid.pdb'.format(DATAPATH)
+                                                    'label': 'Measles nucleocapsid',
+                                                    'value': '{}4uft.pdb'.format(DATAPATH)
+                                                },
+                                                {
+                                                    'label':  'a-cobratoxin-AChBP complex',
+                                                    'value': '{}1yi5.pdb'.format(DATAPATH)
+                                                },
+                                                {
+                                                    'label': 'Calcium ATPase',
+                                                    'value': '{}1su4.pdb'.format(DATAPATH)
                                                 },
                                                 {
                                                     'label': 'DNA',
                                                     'value': '{}1bna.pdb'.format(DATAPATH)
                                                 },
                                                 {
-                                                    'label': 'RNA',
-                                                    'value': '{}6dls.pdb'.format(DATAPATH)
+                                                    'label': 'T7 RNA polymerase',
+                                                    'value': '{}1msw.pdb'.format(DATAPATH)
                                                 },
                                             ],
                                             value='{}1bna.pdb'.format(DATAPATH)
                                         ),
                                     ],
                                 ),
+                                html.Div(
+                                    title='Upload biomolecule to view here',
+                                    className='app-controls-block',
+                                    id='mol3d-upload-container', children=[
+                                        dcc.Upload(
+                                            id='mol3d-upload-data',
+                                            className='control-upload',
+                                            children=html.Div([
+                                                'Drag and drop or click to upload a file.',

```

Comment:
```suggestion
                                                'Drag and drop or click to upload a file',
```
Would need to confirm with a UX expert, but I'm pretty sure we don't need the ending period here.

Reply:
We have the ending period in many other apps -- we can change them all at once in a separate PR.

Reply:
But "Download sample structure" right below and "Select structure" right above, in the same app don't end with a period...